docs/docs/server/introduction/accounts-and-projects.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Accounts and projects
+titleTemplate: :title | Introduction | Server | Tuist
+description: Learn how to create and manage accounts and projects in Tuist.
+---
+
+# Accounts and projects
+
+## Accounts
+
+To use the server, you'll need an account. There are two types of accounts:
+
+- **Personal account:** Those accounts are created automaticaly when you sign up and are identified by a handle that's obtained either from the identity provider (e.g. GitHub) or the first part of the email address.
+- **Organization account:** Those accounts are manually created and are identified by a handle that's defined by the developer. Organizations allow inviting other members to collaborate on projects.
+
+If you are familiar with [GitHub](https://github.com), the concept is similar to theirs, where you can have personal and organization accounts, and they are identified by a *handle* that's used when constructing URLs.
+
+> [!NOTE] CLI-FIRST
+> Most operations to manage accounts and projects are done through the CLI. We are working on a web interface that will make it easier to manage accounts and projects.
+
+You can manage the organization through the sub commands under `tuist organization`.
+
+## Projects
+
+Your projects, either Tuist's or raw Xcode's, need to be integrated with your account through a server-side project. Continuing with the comparison with GitHub, it's like having a local and a remote repository where you push your changes. You can use the `tuist project` to create and manage projects.
```

Comment:
What do you think about referring to `server-side projects` as remote projects or remote Tuist projects?

For example, [here](https://github.com/tuist/tuist/pull/6687/files#diff-2835abcd5971693d7257fe95c3293c5f245faa597e775c40c5def4833dc415d5R9), we could say that a remote project is required instead of "integrating your project with Tuist Server". Just a thought.

Reply:
I am very board with this. Let's be aligned going forward in the terminology used.